Loop/Managers/DeviceDataManager.swift

```diff
@@ -364,6 +370,38 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
         }
     }
 
+    /// NonMySentry Battery Calculation for Alkaline and Lithuim  #141
+    ///
+    /// - parameter currVoltage: Current Voltage Reading from Pump
+    ///
+    public var x22BatteryPercentRemaining : Double = -1
+    
+    private func setBatteryStatusforNonMySentryPumps(currVoltage : Double){
+        var minVoltage : Double
+        var maxVoltage : Double
+        var batteryNotification : Double
+     
+        // if Lithium set min and max linear voltages
+        if (self.batteryChemistry == .lithium){
+            minVoltage = 1.32
+            maxVoltage = 1.58
+            batteryNotification = 0.12
```

Comment:
The point at which we display 0% battery should be equal the point at which a notification should be made, no?